Re-write Readme, creating Firebase hosting setup, enable Google authe… (#2)
OpenTechConsult/social-app-front: README.md

# Build a Popular Social Network with MERN

Welcome ! Here we build an awesome popular social network using the MERN framework.

_Here are the characteristics of the project:_

- The back-end is hosted in Heroku.
- The front-end site is hosted in Firebase.
- Firebase also handles the authentication functionality.
- Material-UI provides the icons in this project. 
- We also use styled components and CSS.
- Pusher is used since MongoDB is not a real-time database like Firebase, and you want the posts to reflect the moment someone hits the submit.

> In this project, you build a social media app that has Google authentication. The app’s look and feel just like the popular Facebook social network. The user can post an image along with descriptive text.

Author: ```suggestion
In this project,  we ou build a social media app that has Google authentication. The app looks and feels just like the popular Facebook social network. The user can post an image along with the descriptive text.
```
